### PR TITLE
(docs) O3-1993: Added missing yarn install command in the readme 

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
This PR adds the missing yarn install command in the setup section of readme.

## Related Issue
https://issues.openmrs.org/browse/O3-1993

### CODE REVIEW

**Comment by ayush-AI** [comment]
Please have a look @ibacher @vasharma05.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,12 @@ In addition to these widgets, two other microfrontends exist that encapsulate cr
 
 Check out the developer documentation [here](http://o3-dev.docs.openmrs.org).
 
-This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna). 
+This monorepo uses [yarn](https://yarnpkg.com) and [lerna](https://github.com/lerna/lerna).
+
+To install the dependencies, run:
+```bash
+yarn install
```
Comment (ibacher) [inline README.md:37]:
```suggestion
yarn
```

**Review by ibacher — APPROVED** [review]
IIRC, this is what we recommend elsewhere. Nice work!

File: README.md (hunk 1 above)
Comment (ayush-AI) [reply README.md]: Thanks @ibacher. I updated the PR as per the suggestions.

**Comment by ayush-AI** [comment]
> IIRC, this is what we recommend elsewhere. Nice work!

Do you mean yarn instead of yarn install?

**Comment by denniskigen** [comment]
@ayush-AI I think what @ibacher is referring to is [this](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/README.md#setup) section of the README that links you to the Dev Docs.

**Comment by ayush-AI** [comment]
> @ayush-AI I think what @ibacher is referring to is [this](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/README.md#setup) section of the README that links you to the Dev Docs. Actually, now that we've mentioned it, it'd probably be nice if that link sent you directly to the [setup](https://o3-dev.docs.openmrs.org/#/getting_started/setup) page of the Dev Docs.

Okay, then I will create another PR for that.

**Pull request merged**